src/main/java/org/joda/time/base/BaseDateTime.java

```diff
@@ -134,6 +134,10 @@ public BaseDateTime(long instant, Chronology chronology) {
         super();
         iChronology = checkChronology(chronology);
         iMillis = checkInstant(instant, iChronology);
+        // validate not over maximum
+        if (iChronology.year().isSupported()) {
+            iChronology.year().set(iMillis, iChronology.year().get(iMillis));
+        }
```

Comment:
I assume this diff is the difference between this class in 2.3 and 2.7, is it correct?